src/Compilers/Server/PortableServer/project.json

```diff
@@ -1,17 +1,31 @@
 ﻿{
-    "dependencies": { },
-    "frameworks": {
-        // We don't actually target netstandard1.6; this is to work around https://github.com/dotnet/roslyn/issues/12458
-        "netstandard1.6": {
-            "imports": "portable-net452"
-        },
-        "NETCoreApp1.0": {
-            "imports": "portable-net452"
-        }
+  "dependencies": {
+    "System.AppContext": "4.1.0",
+    "System.Collections": "4.0.11",
+    "System.Collections.Immutable": "1.2.0",
+    "System.Console": "4.0.0",
+    "System.Diagnostics.Process": "4.1.0",
+    "System.IO.FileSystem": "4.0.1",
+    "System.IO.Pipes": "4.0.0",
+    "System.Linq": "4.1.0",
+    "System.Net.Sockets": "4.1.0",
+    "System.Runtime.Extensions": "4.1.0",
+    "System.Runtime.InteropServices": "4.1.0",
+    "System.Runtime.Loader": "4.0.0",
+    "System.Threading.Thread": "4.0.0"
+  },
+  "frameworks": {
+    // We don't actually target netstandard1.6; this is to work around https://github.com/dotnet/roslyn/issues/12458
+    "netstandard1.6": {
+      "imports": "portable-net452"
     },
-    "runtimes": {
-        "win7-x64": { },
-        "ubuntu.14.04-x64": { },
-        "osx.10.10-x64": { }
+    "NETCoreApp1.0": {
+      "imports": "portable-net452"
```

Comment:
Do we still need to do this?  Entire app top to bottom should now be Net Standard.